### PR TITLE
Templatize imagePullSecrets for NATS

### PR DESCRIPTION
**Pull request by alexplischke**
**What this PR does / why we need it**:
Templatizes the variable _imagePullSecrets_.
The settings were already present in the README and the values.yaml/values-production.yaml file, but the actual template for it was missing.

### CODE REVIEW

File: stable/nats/values.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ image:
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
   ##
   # pullSecrets:
-  #   - myRegistrKeySecretName
+  #   - name: myRegistrKeySecretName
```
Comment (davidkarlsen) [inline stable/nats/values.yaml:18]: please add the option to the README too

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/nats/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: nats
-version: 0.0.3
+version: 0.0.4
```
Comment (davidkarlsen) [inline stable/nats/Chart.yaml]: this needs another bump as some other merge has gone in first: https://github.com/helm/charts/blob/master/stable/nats/Chart.yaml

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/nats/values.yaml (hunk 1 above)
Comment (alexplischke) [reply stable/nats/values.yaml:18]: Funny enough, it was already present in the README, but the actual templated code was missing.

**Pull request merged**